content/en/user-guide/tools/cloud-pods/launchpad/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Cloud Pods Launchpad"
+weight: 10
+categories: ["LocalStack Pro", "Tools"]
+description: >
+  Our Cloud Pods Launchpad offers the easiest way of sharing and injecting cloud pods directly from GitHub.
+---
+
+## Adding a badge to your repository
+
+If you want to add a badge to your README that links to the Cloud Pod Launchpad, you can use the following markdown snippet:
+
+```markdown
+[![LocalStack Pods Launchpad](https://localstack.cloud/gh/pod_badge.svg)](https://app.localstack.cloud/launchpad?url=url_of_your_pod)
```

Comment:
```suggestion
[![Launch LocalStack Cloudpod](https://localstack.cloud/gh/launch-pod-badge.svg)](https://app.localstack.cloud/launchpad?url=url_of_your_pod)
```

Reply:
adjusted the other repos accordingly (had to change the name of the badge file) + updated with a screenshot of a repo + badge